### PR TITLE
Add documentation about RESTFUL_JSON setting

### PR DESCRIPTION
**Pull request by justanr**

### CODE REVIEW

File: docs/extending.rst [hunk 1]
```diff
@@ -40,6 +40,21 @@ object.
...
+    class MyConfig(object):
+        RESTFUL_JSON = {'separators': (', ', ': '),
+                        'indent': 2,
+                        'cls': MyCustomEncoder}
+
+.. Note ::
+
+    When running an application in debug mode, if either ``sort_keys`` or
+    ``indent`` are not declared in the ``RESTFUL_JSON`` configuration setting,
+    Flask-RESTful will provide defaults of ``True`` and ``4`` respectively.
```
Comment (joshfriend) [inline docs/extending.rst]: **IF** the app is in debug mode (`app.debug is True`)

**Comment by justanr** [comment]
@joshfriend I adjusted the wording, if it's good I'll squash them together.

**Comment by joshfriend** [comment]
LGTM.

**Comment by justanr** [comment]
Squashed. :)

**Comment by joshfriend** [comment]
TYVM!

**Pull request merged**